slides/_licenses.qmd

```diff
@@ -0,0 +1,34 @@
+ 
+## How to choose a license
+ 
+ 
+:::: {.columns}
+::: {.column width="70%"}
+ - [https://choosealicense.com/licenses/](https://choosealicense.com/licenses/)
```

Comment:
Could also mention the ability to easily add licenses in GitHub/GitLab?

Reply:
I added screenshots

Reply:
Good idea!